src/server/controllers/event/event.controller.js

```diff
@@ -61,3 +61,24 @@ export const deleteEvent = async (req, res, next) => {
     return next(error);
   }
 };
+
+// implement event search functionality using location, timezone that is start time
+
+export const searchEvent = async (req, res, next) => {
+  try {
+    const { query: { location, startTime } } = req;
+
+    if (location) {
+      const events = await Event.findOne({ location });
```

Comment:
Okay, works now. But the `findOne` returns just one item instead of all. So it should be `findAll` or `find`

Reply:
Yeah, I just discover that now. Let me update it